### PR TITLE
Cleans up tag syntax page

### PR DESCRIPTION
**Pull request by lukaszlenart**

### CODE REVIEW

File: source/tag-developers/if-tag.md [hunk 1]
```diff
@@ -1,92 +1,36 @@
...
+  title: Tag Reference
+  url: tag-reference.html
 ---
 
 # if
 
+Please make sure you have read the [Tag Syntax](tag-syntax) document and understand how tag attribute syntax works.
 
-Please make sure you have read the [Tag Syntax](tag-syntax.html) document and understand how tag attribute syntax works.
+## Description__
```
Comment (JCgH4164838Gh792C124B5) [inline source/tag-developers/if-tag.md]: Looks like a small typo in the If tag generated HTML.  I think `## Description__` should be `## Description` .

The "__" shows up on the page for the staged site.

File: source/tag-developers/textarea-tag.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-layout: default
+layout: tag-developers
 title: Tag Developers Guide
 ---
```
Comment (JCgH4164838Gh792C124B5) [inline source/tag-developers/textarea-tag.md]: The rest of the textarea tag content was _not_ changed in this PR, but when viewed in the staged site it looks _different_ to the other tags in the _tag-reference_ section.  The other tags seemed to have been modified for the description/attribute but not the textarea.

Maybe changing:

`__Description__`  to  `## Description`
`__Parameters__`   to  `## Attributes`

would make textarea consistent with the other tags in _tag-reference_ ?

File: source/tag-developers/if-tag.md (hunk 1 above)
Comment (lukaszlenart) [reply source/tag-developers/if-tag.md]: My bad, fixed :)

File: source/tag-developers/textarea-tag.md (hunk 2 above)
Comment (lukaszlenart) [reply source/tag-developers/textarea-tag.md]: Looks like I had started but didn't finish this page transformation.

**Comment by lukaszlenart** [comment]
> Sorry for the delay in reviewing.
No worries, you did a lot :)

**Pull request merged**